lfs/util.go

```diff
@@ -51,12 +52,14 @@ func (f *GitFilter) CopyCallbackFile(event, filename string, index, totalFiles i
 	}
 
 	var prevWritten int64
-
+	deadline := time.Now().Add(time.Millisecond * 250)
 	cb := tools.CopyCallback(func(total int64, written int64, current int) error {
-		if written != prevWritten {
-			_, err := file.Write([]byte(fmt.Sprintf("%s %d/%d %d/%d %s\n", event, index, totalFiles, written, total, filename)))
+		now := time.Now()
+		if written != prevWritten && (!now.Before(deadline) || written == total) {
```

Comment:
I don't know if it's necessary, but I might be inclined to write `written >= total` just to cover the exceptional case where more bytes are written than expected.  That would, of course, mean the throttling would also stop and we'd write the progress message on every copy callback, but I think that's tolerable as this would be an exceptional condition which may never actually happen.

Reply:
Done